catalogs/catalog-lakehouse-iceberg/src/main/java/com/datastrato/gravitino/catalog/lakehouse/iceberg/IcebergTable.java

```diff
@@ -60,6 +70,40 @@ public CreateTableRequest toCreateTableRequest() {
     return builder.build();
   }
 
+  /** Convert the distribution of Iceberg to the distribution of Gravitino. */
+  private void convertDistribution() {
```

Comment:
suggest to refactor the function like below, we could add a UT to this function.
```java
String IcebergDistributionName =  transformDistribution(GravitinoDistribution xx)
properteis.put(xx, xxx)
```